demos/online-boutique-asm-manifests/authorization-policies/kustomization.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Sample to illustrate how to customize the OnlineBoutique namespace and the Ingress Gateway information, you need to replace:
+# ONLINEBOUTIQUE_NAMESPACE in for-namespace/kustomization.yaml
+# and ONLINEBOUTIQUE_NAMESPACE, INGRESS_GATEWAY_NAMESPACE and INGRESS_GATEWAY_NAME in for-ingress-gateway/kustomization.yaml
+# before deploying this file.
+# As an example:
+# sed -i "s/ONLINEBOUTIQUE_NAMESPACE/onlineboutique/g" for-namespace/kustomization.yaml
+# sed -i "s/ONLINEBOUTIQUE_NAMESPACE/onlineboutique/g;s/INGRESS_GATEWAY_NAMESPACE/asm-ingress/g;s/INGRESS_GATEWAY_NAME/asm-ingressgateway/g" for-ingress-gateway/kustomization.yaml
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+components:
+- all
+- for-ingress-gateway
+- for-namespace
+#- for-memorystore
```

Comment:
There is different `for-memorystore` overlays in `base`, `sidecars` and `authorizationpolicies` folders, indeed. These are for the scenario where users don't want to use the in-cluster `redis` container, they could easily apply these `for-memorystore` overlays to properly configure their OnlineBoutique setup.
This is out-of-scope for the tutorial, but it's a good scenario to test the `Kustomize` approach with folder structure, `resources` versus `components` files, etc.
I could document how to chain this in order to test it as part of this PR, or like you suggested, it will part of a `README.md` file we could add later too.
Does that make sense? Let me know if you need complementary information about that.

Reply:
Yes, this makes sense.
The `for-memorystore` flavour is opt-in — the others are default.